xmlmemory.c

```diff
@@ -978,13 +978,13 @@ xmlInitMemory(void)
      xmlMemInitialized = 1;
      xmlMemMutex = xmlNewMutex();
 
-#ifdef HAVE_STDLIB_H
+#ifdef defined(HAVE_STDLIB_H) && !defined(__ORBIS__)
      breakpoint = getenv("XML_MEM_BREAKPOINT");
      if (breakpoint != NULL) {
          sscanf(breakpoint, "%ud", &xmlMemStopAtBlock);
      }
 #endif
-#ifdef HAVE_STDLIB_H
+#ifdef defined(HAVE_STDLIB_H) && !defined(__ORBIS__)
```

Comment:
#ifdef defined? that's probably a bug, should be `#if defined(...`

Reply:
yep, it's a bug.

Reply:
fixed